test/problems/test_lasso_small_strongly_convex.jl

```diff
@@ -0,0 +1,47 @@
+using LinearAlgebra
+using Test
+using Random
+
+using ProximalOperators
+using ProximalAlgorithms
+
+@testset "Lasso small (strongly convex, $T)" for T in [Float32, Float64]
+
+    Random.seed!(777)
+    dim = 5
+    μf = T(1.0)
+    Lf = T(10.0)
+
+    x_star = convert(Vector{T}, 1.5 * rand(T, dim) .- 0.5)
+
+    lam = convert(T, (μf + Lf) / 2.0)
+    @test typeof(lam) == T
+
+    D = Diagonal(convert(Vector{T}, sqrt(μf) .+ (sqrt(Lf) - sqrt(μf)) * rand(T, dim)))
```

Comment:
I think `convert` can be removed here too

Reply:
Changed. The tests run without any issues.